tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml

```diff
@@ -13,6 +13,7 @@ parameters:
   - 8.2.1.8
   - 8.0.1.6
   - 8.4.1.5
+  - 8.5.1.1
```

Comment:
This PR does not yet set 8.5 as the default. It's only an option. Should we make 8.5 the default now?

Reply:
I think we can wait for TRT 8.5 GA to be merged into main first and then make 8.5 the default.